layouts/partials/helper/get-supporter-list.html

```diff
@@ -0,0 +1,12 @@
+{{ $data := index site.Data.supporters . }}
+
+{{ range sort $data.list "sort" "asc" }}
+  <li>
+    {{ if ne .url "" }}
```

Comment:
This can be `{{ if .url }}`.

Reply:
This is resolved

https://github.com/spotlightpa/poor-richard/pull/338/commits/df040ffa02181793889a73c443e9e36f94e3e7f2